Config/supersid.cfg

```diff
@@ -23,6 +22,11 @@ log_format = supersid_extended
 log_type = filtered
 scaling_factor = 1.0
 
+# fixed y axis for the tk viewer (float value)

```

Comment:
I don't see an explanation of these parameters in ConfigHelp.md.

Reply:
- `log_type` is described in ConfigHelp.md in line 40 with a reasonable description.
- `scaling_factor` is listed in ConfigHelp.md in line 75, unfortunatelly without a description.

scaling factor seems to be untouched since 8, respective 9 years.
https://github.com/sberl/supersid/blame/master/docs/ConfigHelp.md
https://github.com/sberl/supersid/blame/master/supersid/sampler.py

It is used in sampler.py, class Sampler(), method capture_1sec().
```
            # Scale A/D raw_data to voltage here
            # Might substract 5v to make the data look more like SID
            if(self.scaling_factor != 1.0):
                self.data *= self.scaling_factor
```
It is obvious, that the data captured from the sound card via one of the modules alsaaudio, pyaudio or sounddevice is multiplied with this factor. But there is no further description on how to calculate a scaling factor. I doubt this is possible at all after passing the captured data through many software layers with gain adaptation, format transformations, resampling, ... whatever is done in the ALSA stack and the Python sound modules.

If one would have an ADC, would know the schematics, the reference voltage, analog amplifiers, ... whichever parameters are reqiured, one would be able to calculate a voltage from the measured ADC value. I did things like this when developing firmware for automotive devices. Sorry, here we do not know anything about the hardware inside the sound card. Even if we would know, what we receive from the Python sound modules is not what has been captured by the ADC but something that has been processed in the meantime.

Only Eric Gibert knows what the configuration option `scaling_factor` is good for.

Reply:
Sorry if this wasn't clear. I was referring to the psd max min stuff to lock in the y axis. The new parameters that appear after line 25.

Reply:
I'll add this desription to ConfigHelp.md:

> * scaling_factor: float, set it to **1.0**. The data captured from the sound card is multiplied with this value.

Reply:
These are present in ConfigHelp.md lines 78 to 81

>  * psd_min: float, min value for the y axis of the psd graph, **NaN** (default) means automatic scaling
>  * psd_max: float, max value for the y axis of the psd graph, **NaN** (default) means automatic scaling
>  * psd_ticks: int, number of ticks for the y axis of the psd graph, **0** (default) means automatic ticks.

Reply:
Sure. That description of scaling_factor is fine, but it isn't what the original comment was referring to. 
I'd like to see a description of the new config parameters you added:

```
psd_min = NaN
psd_max = NaN
psd_ticks = 0
 
```
added to ConfigHelp.md.